Framework/Algorithms/src/CreateSampleWorkspace.cpp

```diff
@@ -128,6 +128,7 @@ void CreateSampleWorkspace::init() {
                   "Minimum (default) is 1 scan point, which gives a "
                   "non-scanning workspace.");
 
+  declareProperty("InstrumentName", "basic_rect", Direction::Input);
```

Comment:
Why are you adding a new property to `CreateSampleWorkspace`? This seems unrelated to the rest of the work.

Reply:
@rosswhitfield When Jose initially looked at the requirements, he thought that it only applies to HYSPEC. So he wanted to add an instrument name validation step. This was supposed to be the helper to write the test. The algorithm is more general than that, so the name validator is not required. But it might be useful for someone else.

Reply:
OK but this new functionality should then have a unit test and its own release note

Reply:
@jmborr ?

Reply:
I'll get to it

Reply:
@AndreiSavici I included Ross' requests. Can you reapprove, please?